DOCUMENTATION.md

```diff
@@ -0,0 +1,18 @@
+## DevTools Documentation - How To Guide
+The documenatation for Dart & Flutter DevTools lives on [flutter.dev](https://docs.flutter.dev/development/tools/devtools/overview).
+
+Our docs are written and pushed up the the [flutter/website](https://docs.flutter.dev/development/tools/devtools/overview)
+repo by the Flutter DevRel team. Developers working on DevTools should collaborate with Flutter tech writers by following
+the below worflow.
+
+## How to request documenation updates
```

Comment:
This section explains how to document user flows.

Other types of documentation (like glossaries, high level overviews or coding recipes) cannot be illustrated by a video and require tech review first. 

Would it make sense to create another section for them?

Reply:
added another section